Create traditional-bolognese-sauce.md (#278)
LukeSmithxyz/based.cooking: src/traditional-bolognese-sauce.md

# Traditional bolognese sauce

Basic bolognese sauce for lasagne or pasta dishes

- ⏲️ Prep time: 45 min

> - 🍳Cook time: atleast 4 hours
> 
> 

Reviewer: ```suggestion
- 🍳 Cook time: at least 4 hours

```

## Ingredients

- extra virgin olive oil
- 2 carrots
- 2 celery sticks
- 1 large onion
- 2 garlic cloves (optional, but not traditional)
- 200 grams pancetta (optional)
- 300 grams beef mince
- 350 grams pork mince
- 350 grams veal mince (if you cant find veal replace with pork)
- 140 grams concentrated tomato paste 
- 690 grams diced tomatoes
- water
- 1 can peeled tomatoes 
- 1-2 glasses wine (whatever you have open)
- 1 glass of milk (optional)
- 1 glass of beef/vegetable stock (optional)

## Directions

1. Finely dice the carrots, celery, onion and pancetta, if your blender can handle it you can optionally blend all this to speed it up.
2. Heat up a large saucepan or dutch oven on medium low heat and put enough olive oil in to cover the bottom of the pan.

> 3. Put in your diced vegies and pancetta and let it slowly sauté for about 10 minutes until everything is fragrant and soft.

Reviewer: ```suggestion
3. Put in your diced veggies and pancetta and let it slowly sauté for about 10 minutes until everything is fragrant and soft.
```

4. If you are using garlic add it here and let it sweat for a minute
5. Add in your minced meat and break it up into small clumps about the size of a rice grain with a spoon, keep stiring until all your meat is lightly cooked.

> 6. Once all your meat is cooked and looks a bit grey add your wine, add enough to fully deglaze the pan (i usually add a little bit extra for the flavor) 

Reviewer: ```suggestion
6. Once all your meat is cooked and looks a bit grey add your wine, add enough to fully deglaze the pan (I usually add a little bit extra for the flavor) 
```

> stir it through until it has completely evaporated.( if you are using it you can also add the stock at this point)

Reviewer: ```suggestion
stir it through until it has completely evaporated. (If you are using it you can also add the stock at this point)
```

> 6. Once your pan is somewhat dry you can add the tomato paste to the beef, let it cook for a minute to remove the slight metalic taste.

Reviewer: ```suggestion
6. Once your pan is somewhat dry you can add the tomato paste to the beef, let it cook for a minute to remove the slight metallic taste.
```

7. Add your canned and peeled tomatoes and use your spoon to break them up, you don't have to be too thorough since they will soften up while the sauce cooks.
8. add the diced tomatoes, fill the jar your tomatos came in with water give it a shake and add that to the pot, do the same with your can of peeled tomatoes.
9. lower your heat to the lowest setting and slightly cover the pan with your lid.

> 10. let it simmer for as long as you can wait, preferably atleast 4 hours, if it starts getting a bit too dry you can add a bit more water to sure the sauce doesn't burn.

Reviewer: ```suggestion
10. let it simmer for as long as you can wait, preferably at least 4 hours, if it starts getting a bit too dry you can add a bit more water to make sure the sauce doesn't burn.
```

> 11. once its done simmering for as long as you can muster and its nice and thick, the liquid should not come above your meat and should be pretty concentrated

Reviewer: ```suggestion
11. Once it's done simmering for as long as you can muster and its nice and thick, the liquid should not come above your meat and should be pretty concentrated
```

> you can add your glass of milk, let it simmer for 10 more minutes and serve.

Reviewer: ```suggestion
12. You can add your glass of milk, let it simmer for 10 more minutes and serve.
```
